src/dynamoTable.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+
+class dynamoTable(object):
+    
+    """Initialize table
+       Can be buildings, machines, users
+    """
+    def __init__(self, tableType):
+        allowedTableTypes = ['Buildings', 'Machines', 'Users']
+        if tableType not in allowedTableTypes:
+            raise ValueError('Invalid table type! Must be one of: {}'.format(allowedTableTypes))
+        dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+        self.table = dynamodb.Table(tableType)
+
+    def putItem(self, Info):
+        self.table.put_item(Item=vars(Info))
+    
+    def getItem(self, keyInfo, keyStr):

```

Comment:
Neat python trick: You can set up a method to take something called "keyword args". What this means is Python will automatically take what the user calls a value and uses it as a key in a dictionary of arguments. For example:

```
>>> def test(**kwargs):
...     print(kwargs)
... 
>>> test(buildingId='building1')
{'buildingId': 'building1'}
```

So you could do that here and instead of having a user pass in something like `('buildingId', 'building1')`, they could pass in `buildingId='building1'`.

Then, you would make sure that the dictionary only has 1 element, take the key and the value, and use that.